chromadb/utils/embedding_functions.py

```diff
@@ -815,6 +815,61 @@ def __call__(self, input: Documents) -> Embeddings:
         )
 
 
+class OllamaEmbeddingFunction(EmbeddingFunction[Documents]):
```

Comment:
is Ollama EF "just" `HuggingFace Embedding server` under the hood? just trying to square up the few mentions of `HuggingFace Embedding server` here

Reply:
@jeffchuber, Ollama also accepts models (e.g., llama2) and can serve multiple models at the same time (at least from my experience with it). The HuggingFace server serves a single model that has been configured upon container start. 

Another key difference is that Ollama accepts a single text to embed, whereas the HF server accepts batches (whatever the sentence-transformer model accepts in theory). I have created two PRs towards Ollama for accepting batches of texts for embedding (https://github.com/ollama/ollama/pull/2924) and OpenAPI-compliant embedding endpoint (stacked on top of the previous one - https://github.com/ollama/ollama/pull/2925). But so far, there has been no success in getting these merged.

In the end, both are just HTTP endpoints, as are all the other API-based endpoints. If I can merge the PRs, then we can use the OpenAI EF.

Reply:
@tazarov are you saying Yes, you are just using huggingface under the hood, or No and explaining whats different. Looks like your PR came in at the time that Ollama started supporting BERT and Nomic Embed.